app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def new
+
+  end
+
+  def create
+    user = User.new(user_params)
+    @params = user_params
```

Comment:
We might be able to move this to line 13 so that the variable is only set when it's needed.